### PR TITLE
Add wildcard DNS support for nip.io and sslip.io to tests

### PR DESCRIPTION
**Pull request by pfmackin**
- Update the tests to use nip.io as a default wildcard DNS domain. 
- Allow user to select niip.io,  xip.io, or sslip.io
- Fix bug in verify-register that caused a panic

# Checklist 

As the author of this PR, I have:

- [ ] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: ci/scripts/prepare_jenkins_at_environment.sh [hunk 1]
```diff
@@ -16,6 +16,7 @@ if [ -z "$JENKINS_URL" ] || [ -z "$GO_REPO_PATH" ] || [ -z "$TESTS_EXECUTED_FILE
 fi
 
 INSTALL_CALICO=${1:-false}
+WILDCARD_DNS_DOMAIN=${2:-"x=nip.io"}
```
Comment (kminder) [inline ci/scripts/prepare_jenkins_at_environment.sh:19]: What is the x= here?

File: tests/e2e/pkg/dns.go [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+package pkg
+
+import "strings"
+
+// Returns well-known wildcard DNS name is used
+func GetWildcardDNS(s string) string {
+	wildcards := []string{"xip.io", "nip.io", "sslip.io"}
+	for _, w := range wildcards {
+		if strings.Contains(s, w) {
```
Comment (kminder) [inline tests/e2e/pkg/dns.go:12]: Not worth fixing for testware but this should have been an endswith .xip.io, etc. right?

Comment (kminder) [reply tests/e2e/pkg/dns.go:12]: Nevermind.  I see the test case that that suggests otherwise.

File: ci/multicluster/Jenkinsfile [hunk 3]
```diff
@@ -287,7 +291,7 @@ pipeline {
                                     }
                                     steps {
                                         script {
-                                            configureVerrazzanoInstallers(env.INSTALL_CONFIG_FILE_OCIDNS, "./tests/e2e/config/scripts/process_oci_dns_install_yaml.sh")
+                                            configureVerrazzanoInstallers(env.INSTALL_CONFIG_FILE_OCIDNS, "./tests/e2e/config/scripts/process_oci_dns_install_yaml.sh", $ACME_ENVIRONMENT)
```
Comment (bobdonat) [inline ci/multicluster/Jenkinsfile:294]: How is this change related to dns wildcard

Comment (kminder) [inline ci/multicluster/Jenkinsfile:294]: What does this have to do with the DNS change?

**Review by kminder — APPROVED** [review]
Approved.  Paul will fix default param typo in follow up PR.

File: tests/e2e/config/scripts/process_xipio_install_yaml.sh [hunk 4]
```diff
@@ -4,11 +4,13 @@
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
```
Comment (bobdonat) [inline tests/e2e/config/scripts/process_xipio_install_yaml.sh:4]: should name of file be changed to have nipio in the filename since it it not xipio now.  Any references to the file will need changing too.

**Review by bobdonat — APPROVED** [review]

File: tests/e2e/config/scripts/process_xipio_install_yaml.sh (hunk 4 above)
Comment (bobdonat) [reply tests/e2e/config/scripts/process_xipio_install_yaml.sh:4]: Paul will the file name in follow up PR too
